src/js/Game/Game.spec.js

```diff
@@ -3,19 +3,17 @@ import Game from './game';
 describe('Use the game board to mark spaces', () => {
   const gameWithBoard = new Game();
   it('Shows a game board before anything has been selected', () => {
-    expect(gameWithBoard.gameBoard).toStrictEqual([null, null, null, null,
-      null, null, null, null, null]);
+    // eslint-disable-next-line max-len
+    expect(gameWithBoard.gameBoard).toStrictEqual([null, null, null, null, null, null, null, null, null]);
```

Comment:
If possible, I'd try to avoid ignoring linting rules - even on tests. What do you think about something like:
```suggestion
    expect(gameWithBoard.gameBoard).toStrictEqual(
    [null, null, null, null, null, null, null, null, null]
   );
```

or 

```suggestion
    const expectedResult = [null, null, null, null, null, null, null, null, null]
    expect(gameWithBoard.gameBoard).toStrictEqual(expectedResult);
```